inference-engine/src/legacy_api/src/ie_layers.cpp

```diff
@@ -306,6 +306,16 @@ std::string CNNLayer::GetParamAsString(const char* param) const {
     return (*it).second;
 }
 
+std::string CNNLayer::getBoolStrParamAsIntStr(const char *param) const {
+    std::string val = GetParamAsString(param);
+    if (val == "true") {
```

Comment:
Please, add check for "True" and "False" also. Or just convert the string to lowercase always.

@ilyachur , is it acceptable?

Reply:
Yes I think it is acceptable.

Reply:
I will add it, but I'd like to see any example where we could have 'True' or 'False' in CNNLayer's params.

Reply:
Previously MO generated IRs where boolean value could be string "True" or "False". This is why I asked to make this change. Glad to hear that you will change this